creme/reports/templates/reports/view_graph.html

```diff
@@ -1,4 +1,32 @@
 {% extends 'creme_core/detailview.html' %}
+
+{% load i18n creme_widgets reports_tags creme_bricks %}
+{% load blockjsondata get_meta_value from creme_core_tags %}
+
+{% block body_classes %}{% if object.report.is_deleted %}is_deleted{% endif %}{% endblock %}
```

Comment:
Hum I don't think a ReportGraph can be marked as deleted because it's an auxiliary entity which is directly removed.

Reply:
Ok, I just copied this from the previous template, but I agree it does not make sense